crates/bevy_editor_settings/READEME.md

```diff
@@ -0,0 +1,23 @@
+# Bevy Editor Settings
+
+This crate provides a way to have settings for the editor.
+There are three types of settings in order of precedence:
+- User settings
+- Workspace settings
+- Default settings
+
+## User settings
+User settings are settings that are specific to the user. They are stored in the porject at the top level in `user.toml`
```

Comment:
```suggestion
User settings are settings that are specific to the user. They are stored in the project at the top level in `user.toml`
```

Reply:
ya there were a bunch of typos fixed now though